src/django_rich/runner.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import sys
+import unittest
+from unittest.result import STDERR_LINE, STDOUT_LINE
+
+from django.test import testcases
+from django.test.runner import DebugSQLTextTestResult, DiscoverRunner, PDBDebugResult
+from rich.color import Color
+from rich.console import Console
+from rich.rule import Rule
+from rich.style import Style
+from rich.traceback import Traceback
+
+
+class RichMixin(unittest.TextTestResult):
+    # Traceback width defaults to 100
+    console = Console(width=100, stderr=True)
```

Comment:
I think 100 is about right. 88 as default line length (say) + the space needed to get the traceback styling in (10) gets you to ~98 by my counting. 🕵️

Reply:
IMO we shouldn't set the console width, and allow it to be as wide as the user's screen allows, which is the default behaviour?

Reply:
Yup, ok. I was trying to be clever so that if your terminal the `Rules` all match up. But if it's smaller then it will end up looking a bit of a mess if it's fixed at 100. 👍